### PR TITLE
Adding max_message_size property to grpc server.

### PR DESCRIPTION
**Pull request by keshavpeswani**

### CODE REVIEW

File: deployment/terraform/trace-reader/templates/trace-reader.conf [hunk 1]
```diff
@@ -5,13 +5,15 @@ service {
...
     }
+    max.message.size = 52428800 # 50MB in bytes
 }
 
 backend {
   client {
     host = "localhost"
     port = 8090
   }
+  max.message.size = 52428800 # 50MB in bytes
```
Comment (ashishagg) [inline deployment/terraform/trace-reader/templates/trace-reader.conf]: does the grpc client also need this config?

File: reader/src/main/resources/config/base.conf [hunk 2]
```diff
@@ -7,6 +7,7 @@ service {
     cert.path = ""
     private.key.path = ""
   }
+  max.message.size = 52428800 # 50MB in bytes
 }
 
 backend {
```
Comment (ashishagg) [inline reader/src/main/resources/config/base.conf]: same comment applies here. Do we need 'max.message.size' config for the grpc client.

Comment (keshavpeswani) [reply reader/src/main/resources/config/base.conf]: This one is for trace reader server

File: deployment/terraform/trace-reader/templates/trace-reader.conf (hunk 1 above)
Comment (keshavpeswani) [reply deployment/terraform/trace-reader/templates/trace-reader.conf]: Actually this property is a no-op. We can remove it . This was done to keep deployment config and base config in sync.

File: reader/src/main/resources/config/base.conf (hunk 2 above)
Comment (ashishagg) [reply reader/src/main/resources/config/base.conf]: I was pointing to the config within backend. It is not shown in this PR. Check the file.

File: deployment/terraform/trace-reader/templates/trace-reader.conf (hunk 1 above)
Comment (ashishagg) [reply deployment/terraform/trace-reader/templates/trace-reader.conf]: Remove it from both the places.

Comment (keshavpeswani) [reply deployment/terraform/trace-reader/templates/trace-reader.conf]: Yup, I was on it. It is done. Let the build run

**Pull request merged**